src/WebJobs.Script.WebHost/Extensions/ExceptionExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static (string exceptionType, string exceptionMessage, string exceptionDe
 
             string exceptionType = innerException.GetType().ToString();
             string exceptionMessage = Sanitizer.Sanitize(innerException.Message);
```

Comment:
nit: rename `exceptionType` to `innerExceptionType` and rename `exceptionMessage` to `innerExceptionMessage`